components/automate-cli/pkg/verifyserver/server/api/v1/startMockServer.go

```diff
@@ -16,51 +16,34 @@ func (h *Handler) StartMockServer(c *fiber.Ctx) error {
 
 	// If request body is invalid
 	if err != nil {
-		errString := fmt.Sprintf("start mock-server request body parsing failed: %v", err.Error())
-		h.Logger.Error(fmt.Errorf(errString))
-		return fiber.NewError(http.StatusBadRequest, "Invalid request body")
+		h.Logger.Error("Start mock-server request body parsing failed: ", err.Error())
+		return fiber.NewError(fiber.StatusBadRequest, "Invalid request body")
 	}
 
 	// If port number is invalid
-	err = validatePortRange(reqBody.Port, h)
-	if err != nil {
-		h.Logger.Error(err)
-		return fiber.NewError(http.StatusBadRequest, "Invalid port number")
-	}
-
-	// Server is already running in the port
-	if err = validatePortAlreadyInUse(reqBody.Port, h); err != nil {
-		h.Logger.Error(err)
-		return fiber.NewError(http.StatusConflict, fmt.Sprintf(`"%s" server is already running on port %d`, reqBody.Protocol, reqBody.Port))
+	if !isValidPort(reqBody.Port) {
+		h.Logger.Error("Start mock-server request body contains invalid port number")
```

Comment:
Can we simplify this to 

return requestBodyPort >= 0 && requestBodyPort <= 65535

Reply:
Done